CHANGES.txt

```diff
@@ -63,6 +63,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Ninja: Fix issue where Configure files weren't being properly processed when build run
       via ninja.
     - Added ninja mingw support and improved ninja CommandGeneratorAction support.
+    - Made ninja tool force the ninja file as the only target, and make it only update the ninja 
```

Comment:
Seems like this PR is actually 2 PR's?
1 . Fix build.ninja file to always be generated the same (given same build)
2. Do some magic with default targets (which I'm not sure I fully understand yet).

Is that right?

Reply:
Yeah the determinism could be separated

Reply:
@bdbaddog okay this PR now has determinism changes removed.